blocks/library/latest-posts/index.js

```diff
@@ -97,17 +97,56 @@ registerBlockType( 'core/latest-posts', {
 
 		render() {
 			const { latestPosts } = this.state;
-			const { setAttributes } = this.props;
+			const { focus, setAttributes } = this.props;
+			const { displayPostDate, align, layout, columns } = this.props.attributes;
 
-			if ( ! latestPosts.length ) {
-				return (
-					<Placeholder
+			const inspectorControls = (
```

Comment:
Also minor: if this becomes `const inspectorControls = focus && (` then we can reduce one whole indentation level for that assignment.